types/svg/index.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module '*.svg' {
+  import * as React from 'react';
+  const ReactComponent: React.FunctionComponent<React.SVGProps<SVGSVGElement> & { title?: string }>;
+  export default ReactComponent;
+}
```

Comment:
what is this for?

Reply:
Imports of files with the .svg extension in Rollup build are bundled together with the code through libraries like svgr. However, in TypeScript, importing files with the .svg extension is not recognized.

To address this issue, the following code has been added to recognize imports of files with the *.svg extension:
```tsx
import SomeSVG from '../assets/some.svg';

const SomeIcon = () => <SomeSvg />
```

Reply:
nice